packages/app/src/cli/utilities/developer-platform-client.ts

```diff
@@ -0,0 +1,19 @@
+import {PartnersClient} from './developer-platform-client/partners-client.js'
+import {PartnersSession} from '../../cli/services/context/partner-account-info.js'
+import {MinimalOrganizationApp, Organization, OrganizationApp} from '../models/organization.js'
+
+export type Paginateable<T> = T & {
+  hasMorePages: boolean
+}
+
+export function selectDeveloperPlatformClient(): DeveloperPlatformClient {
```

Comment:
I think this name may be confusing, as there's no choice to make, and there are many `selectX` functions that usually prompt the user.

Even when we eventually switch here the client dynamically, it will be something internal and automatic. What about something like `initializeDeveloperPlatformClient`?